impl/pkg/storage/pkarr/pkarr.go

```diff
@@ -0,0 +1,17 @@
+package pkarr
+
+type PkarrRecord struct {
+	// Up to an 1000 byte base64URL encoded string
+	V string `json:"v" validate:"required"`
+	// 32 byte base64URL encoded string
+	K string `json:"k" validate:"required"`
+	// 64 byte base64URL encoded string
+	Sig string `json:"sig" validate:"required"`
+	Seq int64  `json:"seq" validate:"required"`
+}
+
+type PKARRStorage interface {
```

Comment:
I did this but we should rename this to `PkarrStorage` for consistency, since `Pkarr` is how the project is supposed to be represented.

Reply:
naming it storage makes sense to me

Reply:
Went to rename it and i realized this isnt actually used anywhere, so i just removed it. I did rename the only other thing in this file (`PKARRRecord` -> `Record`)